bin/env_variables_all.txt

```diff
@@ -1,8 +1,18 @@
+API_CLIENT_BULK_SCAN_PROCESSOR_ID=bulk_scan_processor
+API_CLIENT_BULK_SCAN_PROCESSOR_SECRET=AAAAAAAAAAAAAAAA
+API_CLIENT_DATA_STORE_ID=ccd_data
+API_CLIENT_DATA_STORE_SECRET=AAAAAAAAAAAAAAAB
```

Comment:
Is there really am IDAM client for `ccd_data`?   (NB: MCA was a special case as it needed its own client so it coudl log in using its service accounts)

Reply:
I don't think we actually need these - have removed them and rerun the case-document-am-api FTA tests and they still pass

Reply:
Fixed in https://github.com/hmcts/ccd-docker/pull/119/commits/889a83335d3eeeb7391dc7c2fb83bd18961410c0